src/pudl/transform/ferc1.py

```diff
@@ -1580,9 +1580,102 @@ def apply_xbrl_calculation_fixes(
                             "name": "electric_operating_revenues",
                             "weight": 1.0,
                             "source_tables": ["electric_operating_revenues_ferc1"],
+                            "utility_type": "electric",
```

Comment:
The utility type seemed to be missing from this calculation. Is that wrong?

Reply:
this one is not strictly necessary to define bc both tables are electric only. as we've discussed at least lightly and I've begun to implement, we will make all of these other-dims explicit in the calculation component table.